<launch>
    <node pkg="ros1_gremsy" type="GimbalNode" name="ros1_gremsy" output="screen">
        <rosparam command="load" file="$(find ros1_gremsy)/config/config.yaml"/>
    </node>
</launch>
